ofrak_core/ofrak/core/elf/unpacker.py

```diff
@@ -115,7 +116,7 @@ async def unpack(self, resource: Resource, config=None):
             e_section_header_r = await resource.create_child(
                 tags=(ElfSectionHeader,),
                 data_range=e_section_header_range,
-                attributes=(ElfSectionStructure.attributes_type(index),),  # type: ignore
+                attributes=(AttributesType[ElfSectionStructure](index),),  # type: ignore
```

Comment:
Are  the `# type: ignore` comments still necessary?

Reply:
I grepped for all the "type: ignore" comments in the lines I made this pattern of change to and removed them